### PR TITLE
Feature/manager

### PR DESCRIPTION
**Pull request by demaceo**
### What does  this PR do?
Creates further functionality for the **Manager** view.
- allows a user to search for past guests.
As well as, 
- correctly registering the specific user upon log in.
- creates a **domObject.js** file to store `querySelectors`.

### Any background context you want to provide?
Working on displaying the searched guest's record data after being clicked on.

### CODE REVIEW

File: src/classes/domObject.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+export const userLoginSection = document.querySelector('.user-login-section');
```
Comment (Estaffieri) [inline src/classes/domObject.js:1]: Yeah!! Nice way to implement this little bit of workflow! Let me know if you found it helpful!

File: src/index.html [hunk 2]
```diff
@@ -13,20 +13,22 @@ <h2 class="hotel-title manager-motto hidden"> <i>Rob the rich</i></h2>
...
   <nav class="manager-nav-bar row-alignment hidden">
-    <a href="#" class="nav-links" id="room-statuses-nav">Room Statuses</a>
-    <a href="#" class="nav-links" id="customer-history-nav">Customer History</a>
-    <a href="#" class="nav-links" id="manage-bookings-nav">Manage Bookings</a>
-    <a href="#" class="nav-links log-out-nav" id="log-out-nav">Log Out</a>
-    <a href="#" class="nav-links"><input class="search-input" type="text" placeholder="Search Users"></a>
+    <section class="nav-search-bar">
+      <input class="search-input" type="text" placeholder="Search Users">
+    </section>
+    <section class="dropdown dropdown-header-icon column-alignment">
```
Comment (Estaffieri) [inline src/index.html:25]: This looks great! Just be mindful that dropdown menus are difficult for screen readers to get into and make tabbing through an application a little more challenging without providing more context(aria-tags), or deliberate focus.(focus-state css)

Comment (demaceo) [reply src/index.html:25]: ooo good point! thanks for that!

**Review by Estaffieri — APPROVED** [review]
Great work here, as usual! Are you running your Lighthouse audits? I'be curious to see how this build performs!

**Pull request merged**